core/src/main/java/io/crate/module/CrateCoreModule.java

```diff
@@ -36,8 +36,7 @@
     public CrateCoreModule(Settings settings, IndexEventListenerProxy indexEventListenerProxy) {
         Logger logger = Loggers.getLogger(getClass().getPackage().getName(), settings);
         this.indexEventListenerProxy = indexEventListenerProxy;
-        if (SharedSettings.ENTERPRISE_LICENSE_SETTING.setting().get(settings) &&
-            "".equals(SharedSettings.LICENSE_IDENT_SETTING.setting().get(settings))) {
+        if (SharedSettings.ENTERPRISE_LICENSE_SETTING.setting().get(settings)) {
```

Comment:
Given that there is no "do we have a valid enterprise license" check here, it could be that this log message is bogus. I think we could remove it altogether since we've the cluster checks.